stacklight_tests/clients/nagios_api.py

```diff
@@ -0,0 +1,124 @@
+import collections
+import urlparse
+
+from stacklight_tests.custom_exceptions import NotFound
+from stacklight_tests import static_elements
```

Comment:
I prefer follow the rules of OpenStack (http://docs.openstack.org/developer/hacking/#imports : we should import only modules instead of classes or other stuff), because there is many reason for it, first of all testing of code(mocking) and readability(many imports on one line)